### ISSUE TITLE
Flickering between connected and disconected non stop

### ISSUE THREAD

**Issue by xGeekpower** [civitai/sd_civitai_extension#37]
Everything was working fine 2 days ago when I first installed the extension and followed the steps, its been stuck like this since.
![2023-03-16 12_59_04- Sidebery  Civitai — Mozilla Firefox](https://user-images.githubusercontent.com/86837667/225735492-13b795c8-aa19-4113-9484-be2f25b798f9.png)

I have SD up to date, I installed python-socketio 5.7.2 as instructed I have no errors on the cmd console when I start up SD.
![2023-03-16 13_19_11-webui-user bat - Shortcut](https://user-images.githubusercontent.com/86837667/225735597-23f4ff01-87c4-4b2b-8042-1b5f9c36ec8d.png)
![2023-03-16 13_33_03-webui-user bat - Shortcut](https://user-images.githubusercontent.com/86837667/225735700-7e826112-d112-4f55-bb0f-37abae8fcad2.png)

Sometimes it actually stay connected for a min or two but when I click to download something from civitai it intermediately disconnects and starts re-connecting and disconnecting non stop, 
![2023-03-16 13_43_16-Select webui-user bat - Shortcut](https://user-images.githubusercontent.com/86837667/225735849-fe3c19f1-d2d4-4707-a2c9-62bf730dc044.png)

I tried to generate new keys and apply on SD settings, just keeps re-connecting and disconnecting non stop, tried reinstalling the extension and same deal.
![2023-03-16 13_22_17- Sidebery  Civitai — Mozilla Firefox](https://user-images.githubusercontent.com/86837667/225734721-3f7ce6ff-88c7-4670-a1a2-b918da32a714.png)

Also tried and deleted my instance to try with a new one and same deal,
![2023-03-16 13_46_36- Sidebery  Civitai — Mozilla Firefox](https://user-images.githubusercontent.com/86837667/225736312-198dc72a-bd80-4771-a016-09a07acbf593.png)

I'm also using citivai helper extension but I don't see any indication that they conflict, so not sure if this is a bug or something on my end.

#######################################################################################################
Initializing Civitai Link
If submitting an issue on github, please provide the below text for debugging purposes:

Python revision: 3.10.9 (tags/v3.10.9:1dd9be6, Dec  6 2022, 20:01:21) [MSC v.1934 64 bit (AMD64)]
Civitai Link revision: c4d4b2e374eccb5f192929a1332e46852f494173
SD-WebUI revision: a9fed7c364061ae6efb37f797b6b522cb3cf7aa2

Checking Civitai Link requirements...
[+] python-socketio[client] version 5.7.2 installed.

#######################################################################################################

**Comment by sraney** [comment]
This was consistent today, I was going to blame Comcast but I'm glad to see others are seeing the same issue.

**Comment by xGeekpower** [comment]
yeah it seems like not a lot of ppl have this issue, I already tried clearing my cookies tried another browser reinstalled the plugin but still cannot connect so probably a connection issue which I have no clue how to solve, whats funny is that whenever is gathering the sha256 the connection is stable but as soon as that ends it disconnects, just hoping a new update comes soon and that would fix this issue.

**Comment by JustMaier** [comment]
Hey everyone. Apologies for missing this. Can you guys give me a little bit more info if you're experiencing this?
- Does it ever connect successfully?
- Does it disconnect and reconnect frequently?
- What browser are you using?
- What version of Automatic are you using? (commit version)
- Are you getting an error message in your console? If so, can you post it here?

**Comment by pryingopen3rdi** [comment]
Yes it connects when I first start up Automatic1111
It disconnects regularly shortly after I start downloading from Civitai
Chrome
updated: 2023-03-29
hash: 22bcc7be
url: https://github.com/AUTOMATIC1111/stable-diffusion-webui/tree/master

**Comment by xGeekpower** [comment]
It does connect successfully but shortly after it will always disconnect 
![2023-04-19 07_41_32- Sidebery  Civitai — Mozilla Firefox](https://user-images.githubusercontent.com/86837667/233094631-b60d5bd7-54d5-423d-a5a0-06710da5a519.png)
![2023-04-19 07_41_45- Sidebery  Civitai — Mozilla Firefox](https://user-images.githubusercontent.com/86837667/233094638-0f0a464a-244c-41d0-8e20-30f112dfae9d.png)

Firefox and tried on Chrome as well same results.
Commit hash: a9fed7c364061ae6efb37f797b6b522cb3cf7aa2
I get no errors at all on the console and my keys never get upgraded unless I restart the whole webui-use.bat which I rather don't do because it takes like 5min to restart :^)
This only worked the day I install it and never again hopefully this works someday.

**Comment by pryingopen3rdi** [comment]
This is still a major issue for me, I signed up as a supporter for civitai link and it doesn't work for me. Is this being worked on, do you need more information?

**Comment by xGeekpower** [comment]
the only times that its stable and connects to the site is when I downloaded a lot of loras/checkpoints that in needs to get the sha256 connection is stable and can even download stuff and see the + and check marks on the thumbnails, but once is done collecting all the sha256 connection drops and cant connect again.
